src/main/java/frc/robot/Constants.java

```diff
@@ -119,11 +119,20 @@ public static final class OIConstants {
     public static final int kDriverControllerPort = 0;
     public static final double kDriverControllerDeadband = 0.15;
     public static final double kDriverControllerTriggerThreshold = 0.7;
+
+    public static final int kOperatorControllerPort = 1;
+    public static final double kOperatorControllerDeadband = 0.15;
+    public static final double kOperatorControllerTriggerThreshold = 0.7;
   }
 
   public static final class AutoConstants {
     public static final double kMaxSpeedMetersPerSecond = 4.5;
     public static final double kPTranslationController = 5;
     public static final double kPRotationController = 5;
   }
+  public static final class armConstants {
+    public static final double controllerGain = 0.03;
+    public static final double feedforwardgain=0.0275;
```

Comment:
1) rename armConstants to ArmConstants
2) rename controllerGain to kP
3) rename feedforwardgain to kG

Reply:
@Cryplo what makes you prefer the shorter variable names in this case?

Reply:
Gains aren't specific and refer to either kP, kI, or kD. Similar for feedforward, gain could refer to kV, kS, etc.